superbuild/projects_modules/qwt.cmake

```diff
@@ -68,15 +69,16 @@ ExternalProject_Add(${ep}
   BINARY_DIR ${build_path}
   TMP_DIR ${tmp_path}
   STAMP_DIR ${stamp_path}
-  SVN_REPOSITORY ${svn_url}
+  GIT_REPOSITORY ${git_url}
+  GIT_TAG ${git_tag}
   CMAKE_GENERATOR ${gen}
   CMAKE_GENERATOR_PLATFORM ${CMAKE_GENERATOR_PLATFORM}
   CMAKE_ARGS ${cmake_args}
   DEPENDS ${${ep}_dependencies}
   UPDATE_COMMAND ""
-  PATCH_COMMAND svn patch ${CMAKE_SOURCE_DIR}/superbuild/patches/qwt-6.3-svn.patch
+  PATCH_COMMAND ${QWT_PATCH_COMMAND}
```

Comment:
There is no patch to apply, unless I did not understand the purpose of forking qwt.

Reply:
yes I will remove it. but ${QWT_PATCH_COMMAND} should be empty now.